mernarezk/assignment2/TreeNode.java

```diff
@@ -0,0 +1,22 @@
+package Assignment2;
+
+/**
+ * simple TreeNode class that takes only int values for simplicity and as both
+ * problems defined for int values.
+ * 
+ * @author Merna
+ *
+ */
+public class TreeNode {
+	int val;

```

Comment:
A good practice for the future would be to make all of these private and final, and only have a public getter method for these attributes.

Reply:
Yea right I will take care of that starting from next assignment